packages/edit-post/src/store/selectors.js

```diff
@@ -236,9 +260,43 @@ export function getActiveMetaBoxLocations( state ) {
  * @return {boolean} Whether the meta box location is active.
  */
 export function isMetaBoxLocationActive( state, location ) {
-	return getActiveMetaBoxLocations( state ).includes( location );
+	return getMetaBoxesPerLocation( state, location ).length !== 0;
```

Comment:
`getMetaBoxesPerLocation` is not guaranteed to return an array, at which point this throws an error.